### DISCUSSION TITLE
Able to add a private PGP key for a reverse-alias to encrypt the emails send to the reverse-alias

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#429] [Feature Requests]
Moved from https://trello.com/c/VEIA1wNd/163-able-to-add-a-private-pgp-key-for-a-reverse-alias-to-encrypt-the-emails-send-to-the-reverse-alias

**Comment by zanona** [comment]
I believe we could also have the ability to encrypt outgoing emails by using a local private key encrypting to signer@simplelogin.io, so SL can decrypt the message and send it unencrypted to the recipient. 
This would be mostly useful when we don't want our email provider to snoop our messages. _(i.e. Gmail)_
At the moment I do have all my incoming messages encrypted by SL along with subject spoofing, but whenever sending emails, it sort of breaks that pattern since those are all readable by Gmail.

Also, a second suggestion would be allowing outgoing messages subject encryption by taking the first line of the encrypted email message, so we could also send messages with an empty subject on Gmail which would then be replaced by the first line of the encrypted email body.

**Comment by vovalev** [comment]
This will be extremely useful. Hope it is possible to implement.

**Comment by ghost** [comment]
Is there any progress?

**Comment by Xynonners** [comment]
any progress?
